rust-runtime/aws-smithy-types/src/instant/mod.rs

```diff
@@ -38,24 +42,54 @@ pub struct Instant {
 
 impl Instant {
     /// Creates an `Instant` from a number of seconds since the Unix epoch.
-    pub fn from_epoch_seconds(epoch_seconds: i64) -> Self {
+    pub fn from_secs(epoch_seconds: i64) -> Self {
         Instant {
             seconds: epoch_seconds,
             subsecond_nanos: 0,
         }
     }
 
+    /// Converts number of milliseconds since the Unix epoch into an `Instant`.
```

Comment:
nit: this doc is inconsistent with the others that start with `Creates an `Instant`